namespaces/live.cloud-platform.service.justice.gov.uk/visit-someone-in-prison-backend-svc-preprod/resources/rds.tf

```diff
@@ -15,10 +15,8 @@ module "visit_scheduler_rds" {
   db_engine_version           = "15.7"
   rds_family                  = "postgres15"
   db_instance_class           = "db.t4g.small"
-  db_max_allocated_storage    = "16300"
-  db_allocated_storage        = "16000"
-  db_iops                     = "12000"
-  db_password_rotated_date    = "2023-03-22"
+  db_max_allocated_storage    = "100"
```

Comment:
Need to ensure you have a prometheus alert setup so the team receives slack/email notifications, as won't receive the generic aws-account level alerts that go to CP team

Reply:
or just not set it at all until post-migration